core/src/main/java/org/apache/spark/util/SerializableConfigurationSuite.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.spark.util;
+
+/**
+ * This test ensures that the API we've exposed for SerializableConfiguration is usable
+ * from Java. It does not test any of the serialization it's self.
+ */
+class SerializableConfigurationSuite {
```

Comment:
Although this is a compilation test, a test suite should not be under `core/src/main/java`.
@HyukjinKwon . Could you make a follow up?

Reply:
Otherwise, this will be a part of our `core` library.

Reply:
Wait, I missed it too. yea will fix it

Reply:
Here https://github.com/apache/spark/pull/25867

Reply:
Thanks!

Reply:
Thanks for catching this all, my bad I thought was in the test directory but didn't pay enough attention.